dhis-2/dhis-api/src/main/java/org/hisp/dhis/visualization/Visualization.java

```diff
@@ -1025,6 +1028,30 @@ public void setYearlySeries( List<String> yearlySeries )
         this.yearlySeries = yearlySeries;
     }
 
+    /**
+     * Returns the list of DimensionDescriptor held internaly to the current Visualization object.
+     * See {@link #holdDimensionDescriptor}.
+     *
+     * @return the list of DimensionDescriptor's held.
+     */
+    public List<DimensionDescriptor> getDimensionDescriptors()
+    {
+        return dimensionDescriptors;
+    }
+
+    /**
+     * This method will hold the mapping of a dimension and its respective formal
+     * type.
+     *
+     * @param dimension: the dimension, which should also be found in
+     *        "{@link #columnDimensions}" and "{@link #rowDimensions}".
+     * @param dimensionType: the formal dimension type. See {@link DimensionType}
+     */
+    public void holdDimensionDescriptor( final String dimension, final DimensionType dimensionType )
```

Comment:
Maybe rename to `addDimensionDescriptor` to make it similar to other naming in the system.

Reply:
Done